src/java.desktop/unix/native/libawt/awt/awt_LoadLibrary.c

```diff
@@ -43,6 +43,12 @@
 #define VERBOSE_AWT_DEBUG
 #endif
 
+#define CHECK_EXCEPTION_FATAL(env, message) \
+    if ((*env)->ExceptionCheck(env)) { \
+        (*env)->ExceptionDescribe(env); \
```

Comment:
Is it possible that `ExceptionDescribe` could leak something important?

Should we preserve `ExceptionClear` instead?